src/components/ContactsList.js

```diff
@@ -1,28 +1,68 @@
-import { useState } from "react"
+import { useState, Fragment } from "react"
```

Comment:
Interesting, imported a Fragment, I would be interested to know what this is used for typically. I've noticed the syntax on a lot of REACT official documentation.

Reply:
Fragments let you add multiple elements to a component without having an extra DOM node created by something like a div. I could have put this in a div and it would have been fine, but I wanted to use fragment for the practice.

Sometimes using divs could cause styling issues with flexbox and grid so this container avoids that.